CHANGES/1245.misc

```diff
@@ -0,0 +1 @@
+``CollectionVersionSignature`` data is now stored as plain text instead of bytes.
```

Comment:
Isn't this a bugfix?

Do you think we need a data-repair command?

Reply:
It hasn't shipped with anything yet, so this should be fine.

Reply:
So do we need to backport any of this?

Reply:
I believe not. This is really a followup to https://github.com/pulp/pulp_ansible/pull/845 .